AEPIdentity/Tests/IdentityStateTests.swift

```diff
@@ -639,6 +639,29 @@ class IdentityStateTests: XCTestCase {
         XCTAssertTrue(mockHitQueue.queuedHits.isEmpty) // hit should NOT be queued in the hit queue
     }
 
+    /// Tests that a hit is queued when forceSynce
+    func testSyncIdentifiers_whenForceSync_shouldQueueHit() {
```

Comment:
Let's also add functional tests checking that shared states are updated as expected and the force sync network request is made on boot, on both install and subsequent launch scenarios.